install/kubernetes/templates/istio-ingress.yaml.tmpl

```diff
@@ -70,6 +70,26 @@ spec:
         ports:
         - containerPort: 80
         - containerPort: 443
+        readinessProbe:
+          exec:
+            command:
+            - /bin/sh
+            - -c
+            - curl http://localhost:15000/clusters && curl http://localhost:15000/listeners && curl http://localhost:15000/routes
+          initialDelaySeconds: 3
+          periodSeconds: 5
+          timeoutSeconds: 3
+          failureThreshold: 10
+        livenessProbe:
+          exec:
+            command:
+            - /bin/sh
+            - -c
+            - /usr/bin/curl http://localhost:15000/clusters
```

Comment:
Can we be consistent  with `/usr/bin/curl` or `curl`.

Reply:
doesn't need sh -c and should directly curl _or_ should actually check for the http return code from the url and not just that it can connect (if envoy returns a 503 it will still "pass")

all in all I think liveness/readyness for envoy/pilot needs some thought and work not trying to rush something without a concrete case (envoy crashing is supposed to be detected by pilot-agent; adding ingresses/scaling (where readyness would be necessary)) is a 0.3 feature